docs/voice/telephony-session-notifications.md

```diff
@@ -0,0 +1,225 @@
+# Telephony Session Notification
+
+A developer can use Telephony Session Notification to detect all changes in the status of a call effectively. 
+
+## What is Telephony Session Notification?
+
+A Telephony Session Notification (Telephony Session Notification) is a series of notifications or events a developer can use to track an end-to-end telephony call in RingCentral. The call can be from an external PSTN or any mobile number to a RingCentral extension, or it can be from a RingCentral Digital Line to an external PSTN number, or it can be between two RingCentral extensions. The Telephony Session Notification provides detailed end-to-end call sessions and party (endpoints/user) level information to enable developers to take specific actions or build custom reports based on that call information. The Telephony Session Notification events can be subscribed both at an Account level or at an Extension level.
```

Comment:
No need to put same term in parentheses. Please remove parentheses and term or change the term in parentheses.

> Telephony Session Notification (Telephony Session Notification)